spring-integration-hazelcast/src/test/java/org/springframework/integration/hazelcast/metadata/HazelcastMetadataStoreTests.java

```diff
@@ -0,0 +1,162 @@
+package org.springframework.integration.hazelcast.metadata;
+
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.IMap;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+
+import static org.mockito.Mockito.*;
+
+
+import org.springframework.integration.metadata.MetadataStoreListener;
+
+import static org.junit.Assert.*;
```

Comment:
Please, don't use asterisk. There is also some kind of trap when the class is imported from the unexpected package.